tests/python/unit/test_binary_model.py

```diff
@@ -0,0 +1,44 @@
+import sys
+sys.path.append("..")
+print('\n'.join(sys.path))
+from generalized_linear_models.dku_glm import BinaryClassificationGLM
+import statsmodels.api as sm
+import unittest
+from testing_utils import testing_dict_binary
+import numpy as np
+import pandas as pd
+from sklearn import datasets
+
+def test_link_function():
+    case = unittest.TestCase()
```

Comment:
Nitpick: you could use pytest fixtures for that. Generally speaking for plugins, we use pytest for everything. It's OK to use other frameworks. I am personally not used to unittest, and I don't know about other developers.